.github/workflows/ruby-ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Ruby-CI
+
+on:
+  push:
+  branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ruby-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout
+
+      - uses: ruby/setup-ruby@v1
+        name: Use Ruby 2.6
+        with:
+          ruby-version: 2.6
+          bundler-cache: true
+      - name: 📦 Install dependencies
+        run: |
+          bundle config set ignore_messages true
```

Comment:
ignores the post install message ..."HEADS UP! i18n 1.1 changed fallbacks to exclude default locale....", example [here](https://github.com/Shopify/shopify_api/pull/815)